base/actions/utils.py

```diff
@@ -2,23 +2,20 @@
 
 
 def get_risk_level(data: Dict[Any, Any]) -> Text:
+    if data.get("exposure") != "no":
+        return "high"
```

Comment:
This seems extreme, if they answer "maybe" they're immediately high risk. Just want to make sure that's what the algorithm actually is.

Reply:
Yes, this was weird for me too, but I confirmed it.